push/sender/src/test/java/org/jboss/aerogear/unifiedpush/test/archive/UnifiedPushArchiveImpl.java

```diff
@@ -65,9 +73,18 @@ public UnifiedPushArchive withMessaging() {
             .addPackage(org.jboss.aerogear.unifiedpush.message.holder.AbstractMessageHolder.class.getPackage())
             .addPackage(org.jboss.aerogear.unifiedpush.message.exception.MessageDeliveryException.class.getPackage())
             .addClasses(AbstractJMSMessageProducer.class, AbstractJMSMessageListener.class, AbstractJMSMessageConsumer.class)
-            .addClasses(AbstractJMSTest.class, JmsClient.class)
+            .addClasses(AbstractJMSTest.class, JmsClient.class, CdiJmsBridge.class)
             .addClasses(DispatchToQueue.class, Dequeue.class)
-            .addAsWebInfResource("hornetq-jms.xml");
+            .addAsWebInfResource(new File("../../servers/ups-wildfly/src/main/webapp/WEB-INF/hornetq-jms.xml"));
+    }
+
+    @Override
+    public UnifiedPushArchive withMessageDrivenBeans() {
+        return addClasses(AbstractJMSMessageListener.class)
+                .addClasses(MessageHolderWithVariantsConsumer.class, MessageHolderWithVariantsProducer.class)
+                .addClasses(MessageHolderWithTokensConsumer.class, MessageHolderWithTokensProducer.class)
+                .addClasses(TriggerMetricCollectionConsumer.class)
+                .addAsWebInfResource(new File("../../servers/ups-wildfly/src/main/webapp/WEB-INF/jboss-ejb3.xml"), "jboss-ejb3.xml");
```

Comment:
These paths and file names should be extracted as constants, specially if they are reused in more places.